src/pyfmi/master.pyx

```diff
@@ -272,11 +272,17 @@ class MasterAlgOptions(OptionBase):
             with an error controlled simulation.
             Default: 0.0 (i.e. inactive)
 
+        local_rtol --
+            Defines the relative tolerance that will be provided to the 
+            connected FMUs during initialization of the underlying 
+            models. Note, only have an effect if 'initialize' is set to
+            True.
+            Default: 1e-6
+
         result_file_name --
             Specifies the name of the file where the simulation result is
-            written. Setting this option to an empty string results in a default
-            file name that is based on the name of the model class.
-            Default: Empty string
+            written. Note that there should be one name for each model.
+            Default: Model name + "_result." + filetype
```

Comment:
It should be specified here what format result_file_name should be passed in as since there is one name for each model.

Reply:
Updated description.